hash/_wasm/build.ts

```diff
@@ -1,48 +1,88 @@
+#!/usr/bin/env -S deno run --allow-all
 // Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
 
-import { encode as base64Encode } from "../../encoding/base64.ts";
+import * as base64 from "../../encoding/base64.ts";
 
-// 1. build wasm
-async function buildWasm(path: string) {
-  const cmd = [
-    "wasm-pack",
+const home = Deno.env.get("HOME");
+const root = new URL(".", import.meta.url).pathname;
+
+if (new URL(import.meta.url).protocol === "file:") {
+  // Run in the same directory as this script is located.
+  Deno.chdir(root);
+} else {
+  console.error("build.ts can only be run locally (from a file: URL).");
+  Deno.exit(1);
+}
+
+// 1. Build WASM from Rust.
+const cargoStatus = await Deno.run({
+  cmd: [
+    "cargo",
     "build",
-    "--target",
-    "web",
     "--release",
-    "-d",
-    path,
-  ];
-  const builder = Deno.run({ cmd });
-  const status = await builder.status();
-
-  if (!status.success) {
-    console.error(`Failed to build wasm: ${status.code}`);
-    Deno.exit(1);
-  }
-}
+    "--target",
+    "wasm32-unknown-unknown",
+  ],
+  env: {
+    // eliminate some potential sources of non-determinism
```

Comment:
I don't think this mitigation works, I still get changes locally after building the latest version of this PR

Reply:
https://github.com/denoland/deno_std/pull/999/files/fa119991d79f7e432111b0546dda24fb06105f88..bae5ded329ea07f4bc062552e6d02d4c10b7253f

Sorry, typo. Fixed.